doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -367,7 +351,7 @@ gls{SoMoX}
 
 \end_inset
 
-, such that it is automatically executed after 
+ to be executed after 
```

Comment:
I think “automatically” is still important. Can you change it to “to automatically be executed after […]”?